po/be.po

```diff
@@ -0,0 +1,581 @@
+
+#: src/models/sockets.js:37
+msgid "X11 windowing system"
+msgstr "Аконная сістэма X11"
+
+#: src/models/sockets.js:44
+msgid "Wayland windowing system"
+msgstr "Аконная сістэма Wayland"
+
+#: src/models/sockets.js:51
+msgid "Fallback to X11 windowing system"
+msgstr "Зварот да аконнай сістэмы X11"
+
+#: src/models/sockets.js:58
+msgid "PulseAudio sound server"
+msgstr "Гукавы сервер PulseAudio"
+
+#: src/models/sockets.js:65
+msgid "D-Bus session bus"
+msgstr "Сесійная шына D-Bus"
+
+#: src/models/sockets.js:72
+msgid "D-Bus system bus"
+msgstr "Сістэмная шына D-Bus"
+
+#: src/models/sockets.js:79
+msgid "Secure Shell agent"
+msgstr "Агент SSH"
+
+#: src/models/sockets.js:86
+msgid "Smart cards"
+msgstr "Смарт-карты"
+
+#: src/models/sockets.js:93
+msgid "Printing system"
+msgstr "Сістэма друку"
+
+#: src/models/sockets.js:100
+msgid "GPG-Agent directories"
+msgstr "Каталогі GPG-агента"
+
+#: src/models/sockets.js:125
+msgid "List of well-known sockets available in the sandbox"
+msgstr "Спіс вядомых сокетаў, даступных у пясочніцы"
+
+#: src/models/variables.js:40
+msgid "Variables"
+msgstr "Пераменныя"
+
+#: src/models/variables.js:43
+msgid "e.g. GTK_DEBUG=interactive"
+msgstr "напрыклад GTK_DEBUG=interactive"
+
+#: src/models/variables.js:73
+msgid "List of variables exported to the application"
+msgstr "Спіс пераменных, якія экспартуюцца ў праграму"
+
+#: src/widgets/aboutDialog.js:38
+msgid "Contributions by"
+msgstr "Уклады ў праект"
+
+#. TRANSLATORS: <full-month-name> <day-of-month>, <year-with-century>
+#: src/widgets/appInfoViewer.js:61
+msgid "%B %e, %Y"
+msgstr "%B %e, %Y"
+
+#: src/widgets/appInfoViewer.ui:127
+msgid "Version"
+msgstr "Версія"
+
+#: src/widgets/appInfoViewer.ui:142
+msgid "Last Updated"
+msgstr "Апошняе абнаўленне"
+
+#: src/widgets/appInfoViewer.ui:157
+msgid "Runtime"
+msgstr "Рантайм"
+
+#: src/widgets/busNameRow.js:35 src/widgets/pathRow.js:73
+#: src/widgets/relativePathRow.js:33 src/widgets/variableRow.js:33
+msgid "This is not a valid option"
+msgstr "Гэта недапушчальны варыянт"
+
+#: src/widgets/detailsButton.js:59
+msgid "_Show Details"
+msgstr "_Паказаць дэталі"
+
+#: src/widgets/detailsButton.js:107
+msgid "Show application in a software manager"
+msgstr "Паказаць праграмы ў кіраўніку праграм"
+
+#: src/widgets/detailsButton.js:109
+msgid "No software manager found"
+msgstr "Кіраўнік праграм не знойдзены"
+
+#: src/widgets/globalRow.ui:10 src/widgets/globalInfoViewer.ui:69
+msgid "All Applications"
+msgstr "Усе праграмы"
+
+#: src/widgets/globalInfoViewer.ui:86
+msgid "Changes that apply to all Flatpak applications"
+msgstr "Змены, якія прымяняюцца да ўсіх праграм Flatpak"
+
+#: src/widgets/globalInfoViewer.ui:153
+msgid "Flatpak Version"
+msgstr "Версія Flatpak"
+
+#: src/widgets/globalInfoViewer.ui:168
+msgid "Portal Version"
+msgstr "Версія партала"
+
+#: src/widgets/globalInfoViewer.ui:183
+msgid "Changes Path"
+msgstr "Шлях змен"
+
+#: src/widgets/menu.ui:7
+msgid "_Help"
+msgstr "_Даведка"
```

Comment:
You need to "mark" different characters for the shortcuts, otherwise you can access the second one (it becomes ambiguous)

![image](https://user-images.githubusercontent.com/481418/215462392-ade5ea9a-5d42-4fa6-9ae4-6b764eaf9749.png)